aiohttp/http_writer.py

```diff
@@ -130,6 +130,7 @@ class PayloadWriter(AbstractPayloadWriter):
     def __init__(self, stream, loop, acquire=True):
         self._stream = stream
         self._transport = None
+        self._buffer = []
```

Comment:
We just follow asyncio streams design, it switched from bytearrays to lists and back to bytearrays :)